GETTING_STARTED_WITH_CELESTIA.md

```diff
@@ -0,0 +1,293 @@
+```
+
+### Setting Up a Celestia Light Node
+
+For this tutorial, we'll use the [Mocha testnet](https://docs.celestia.org/how-to-guides/mocha-testnet). 
+You'll need to run a Celestia light node to connect your rollup to the network.
+
+**Note**: For production deployments, it's recommended to connect your light node to a reliable RPC provider or use a bridge node for enhanced reliability and performance.
+
+#### Installation and Setup
+
+Follow the Celestia documentation for detailed instructions:
+- [Install celestia-node](https://docs.celestia.org/how-to-guides/celestia-node)
+- [Setting up a Celestia light node](https://docs.celestia.org/how-to-guides/light-node)
+
+#### Optimizing Initial Sync
+
+To speed up the initial synchronization, you can configure your light node to start from a recent block:
+
+1. Visit the block explorer: https://mocha.celenium.io/
+2. Select a recent block and note its hash and height
+3. Update your light node configuration:
+   - `Header.TrustedHash`: Use the block hash from step 2
+   - `DASer.SampleFrom`: Use the block height from step 2
+
+**Important**: This optimization means you won't be able to start your rollup from blocks prior to the selected height.
+
+### Preparing Your Node
+
+#### Getting Your Node Address
+
+Use the `cel-key` utility to list your node's address:
+Assuming you're running it from the repo folder after building.
+
+```bash
+$ ./cel-key list --keyring-backend test \
+    --node.type light --p2p.network mocha
+using directory:  /Users/developer/.celestia-light-mocha-4/keys
+- address: celestia1qd73x7lzh97uxm9lxe49qdfmuup25kp4khaxdd
+  name: my_celes_key
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A2hgY3ckADmUQRO01L4J54tZhhZrfE2oGGjGV+63DJcB"}'
+  type: local
+```
+
+#### Funding Your Node
+
+Your Celestia node address needs `TIA` tokens to submit data. For the Mocha testnet, request tokens from the [faucet](https://docs.celestia.org/how-to-guides/mocha-testnet#mocha-testnet-faucet).
+
+### Verifying Your Light Node
+
+Ensure your light node is running and fully synced by checking the sampling statistics:
+
+```bash
+$ celestia das sampling-stats
+{
+  "result": {
+    "head_of_sampled_chain": 7413765,
+    "head_of_catchup": 7413794,
+    "network_head_height": 7413794,
+    "workers": [
+      {
+        "job_type": "recent",
+        "current": 7413767,
+        "from": 7413767,
+        "to": 7413767
+      },
+      {
+        "job_type": "catchup",
+        "current": 7413766,
+        "from": 7413672,
+        "to": 7413767
+      },
+      {
+        "job_type": "recent",
+        "current": 7413768,
+        "from": 7413768,
+        "to": 7413768
+      }
+    ],
+    "concurrency": 3,
+    "catch_up_done": false,
+    "is_running": true
+  }
+}
+```
+
+Key indicators to check:
+- `catch_up_done`: Should be `true` when fully synced
+- `network_head_height`: Current height of the network
+- `head_of_sampled_chain`: Should be close to network height when synced
```

Comment:
if there is no alternative to check whether our light node has synced, lets remove the `head_of_sampled_chain` as a key indicator to check, since it has no use for our customers.